CoreScriptsRoot/Modules/VR/VirtualKeyboard.lua

```diff
@@ -186,6 +187,40 @@ local MINIMAL_KEYBOARD_LAYOUT_SYMBOLS = HttpService:JSONDecode([==[
 ---------------------------------------- END KEYBOARD LAYOUT --------------------------------------
 
 
+local VOICE_STATUS_CODE_ENUM = {}
+do
+	local STATUS_CODES =
+	{
+	    'ASR_STATUS_OK',
+	    'ASR_STATUS_CANCELLED',
+	    'ASR_STATUS_UNKNOWN',
+	    'ASR_STATUS_INVALID_ARGUMENTS',
+	    'ASR_STATUS_DEADLINE_EXCEEDED',
+	    'ASR_STATUS_NOT_FOUND',
+	    'ASR_STATUS_ALREADY_EXISTS',
+	    'ASR_STATUS_PERMISSION_DENIED',
+	    'ASR_STATUS_UNAUTHENTICATED',
+	    'ASR_STATUS_RESOURCE_EXHAUSTED',
+	    'ASR_STATUS_FAILED_PRECONDITION',
+	    'ASR_STATUS_ABORTED',
+	    'ASR_STATUS_OUT_OF_RANGE',
+	    'ASR_STATUS_UNIMPLEMENTED',
+	    'ASR_STATUS_INTERNAL',
+	    'ASR_STATUS_UNAVAILABLE',
+	    'ASR_STATUS_DATA_LOSS',
+	    -- last official google response
+
+	    -- Roblox statuses
+	    'ASR_STATUS_NOT_ENABLED',
+	    'ASR_STATUS_LOW_CONFIDENCE',
+	    'ASR_STATUS_INVALID_JSON'
+	};
+
+	for i, code in pairs(STATUS_CODES) do
```

Comment:
Think we should use ipairs or a for i = 1, style loop, I believe pairs does not guarantee order, even for array style tables.